packages/shared/lib/services/environment.service.ts

```diff
@@ -143,10 +148,12 @@ class EnvironmentService {
             q.where('secret_key_hashed', hash);
         } else if ('publicKey' in opts) {
             q.where('_nango_environments.public_key', opts.publicKey);
-        } else if ('accountId' in opts) {
-            q.where('_nango_environments.account_id', opts.accountId).where('_nango_environments.name', opts.envName);
         } else if ('accountUuid' in opts) {
             q.where('_nango_accounts.uuid', opts.accountUuid).where('_nango_environments.name', opts.envName);
+        } else if ('envName' in opts) {
```

Comment:
can we use `accountId` which would make it immune to ordering change?

Reply:
yes definitely

Reply:
changed